cmd/litefs-bench/main.go

```diff
@@ -53,6 +58,11 @@ func run(ctx context.Context) error {
 	}
 	defer func() { _ = db.Close() }()
 
+	// Initialize cache.
+	if _, err := db.Exec(fmt.Sprintf(`PRAGMA cache_size = %d`, cacheSize)); err != nil {
+		return fmt.Errorf("set cache size to %d: %w", *cacheSize, err)
+	}
+
 	// Set journal mode, if set. Otherwise defaults to "DELETE" for new databases.
 	if *journalMode != "" {
 		if _, err := db.Exec(`CREATE TABLE IF NOT EXISTS t (id INTEGER PRIMARY KEY, num INTEGER, data TEXT)`); err != nil {
```

Comment:
I think this should be something like this:

```go
if _, err := db.Exec(`PRAGMA journal_mode = ` + *journalMode); err != nil {
```

Reply:
Yep. That's definitely a 🤦  :) 

I pushed the fix to https://github.com/superfly/litefs/pull/201. So I far I've just been testing and debugging some `DELETE` journal mode issues so I haven't his this dumb code yet. 😂

Reply:
I tagged you as a coauthor on the commit too. Thanks for commenting on this PR. :)